src/templates/dictatorship/index.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable react/prop-types */
+// THE PEOPLE ASKED FOR IT... 2020/04/01
+import React from 'react'
+import {
+  ClaimDomainScreen,
+  KnownAppBadge,
+  ReviewScreen,
+  TokensScreen,
+} from '../kit'
+import DictatorScreen from './components/DictatorScreen'
+
+import header from './header.svg'
+import icon from './icon.svg'
+
+function completeDomain(domain) {
+  return domain ? `${domain}.aragonid.eth` : ''
+}
+
+export default {
+  id: 'dictatorship-template.open.aragonpm.eth',
+  name: 'Dictatorship',
+  new: true,
+  header,
+  icon,
+  description: `
+    From Hungary, to Russia, to China, to Venezuela: Dictatorship is the
+    new Democracy. Introducing the Aragon Dictatorship template.
+  `,
+  userGuideUrl: 'https://en.wikipedia.org/wiki/Human_rights_in_North_Korea',
```

Comment:
LOL. PD: [Can't stop hearing this in my head](https://www.youtube.com/watch?v=U06jlgpMtQs)